src/InjectPinoLogger.ts

```diff
@@ -26,3 +26,7 @@ export function createProvidersForDecorated(): Array<Provider<PinoLogger>> {
     createDecoratedLoggerProvider(context)
   );
 }
+
+export function getLoggerToken(context: string): string {
+  return `${decoratedTokenPrefix}${context}`;
+}
```

Comment:
Can we export this function from `index` file also and add just a few words on this in docs?

Reply:
Done, thanks.